scripts/st2bootstrap-deb.sh

```diff
@@ -474,8 +474,9 @@ install_st2mistral() {
   # Setup Mistral DB tables, etc.
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
 
-  # Register mistral actions
-  /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
+  # Register mistral actions. Init of git is added here to temporarily fix
+  # a bug in one of the OpenStack dependency related to pbr version checking.
+  git init && /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate && rm -rf .git
```

Comment:
Yes, my plan this week is to submit a patch to make the action registration warn instead of fail for a bad dependency.  I can implement what you suggested as well.